### DISCUSSION TITLE
Invitation: Monday June 24, 2024 5:00PM PDT: DataFusion Meetup in San Francisco

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10800] [General]
Follow on ot https://github.com/apache/datafusion/discussions/10340

# Invitation to the first San Francisco Bay Area Apache DataFusion meetup!

​Come meet with the Apache DataFusion community and learn about the amazing work that people are doing using Apache DataFusion as well as some words about the future of the project.

Kudos to @emgeee and @cpard and @andygrove  for helping organize

# Register

Register here:  https://lu.ma/6bphole2

**Comment by vaibhawvipul** [comment]
It will be great, if we can record it! :)

**Reply by alamb** [reply]
We will try!

**Comment by hveiga** [comment]
Hi folks, it seems the event is already full. How likely it is that some of us in the waitlist will be able to attend? Would it be worth getting there in case some people does not show up? Regardless, it is super cool to see the community getting together. Thank you for organizing this!

**Reply by alamb** [reply]
I think it is better to ask on the luma event page: https://lu.ma/6bphole2

**Reply by cpard** [reply]
hey @hveiga we will keep sending reminders on lu.ma for people to update their RSVP status. It's amazing to experience the interest of people in attending the event, if the community sees value in it, we can definitely keep making these events happen!

**Comment by alamb** [comment]
![PXL_20240625_005835466](https://github.com/apache/datafusion/assets/490673/66822a06-7d10-4971-9282-d50c62f5b83f)
![PXL_20240625_005833618](https://github.com/apache/datafusion/assets/490673/280071ac-b6cd-460e-b575-c5f5b0ede825)
![PXL_20240625_005832190 MP](https://github.com/apache/datafusion/assets/490673/2d255197-6b89-444a-8947-38954c7eda01)
